composer.json

```diff
@@ -7,15 +7,14 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.1",
-        "drupal/core": "~8.6@alpha",
+        "openeuropa/drupal-core-require": "^8.6",
```

Comment:
Here version is good but then "openeuropa/drupal-core-require-dev" should also be "^8.6".
It should appear in alphabetical order.

Reply:
Fixed.